doc/source/images/gen_density_graphic.py

```diff
@@ -12,8 +12,9 @@
 ax.set_xticks(x[1:4])
 ax.set_yticks(y[1:4])
 fontdict = {'fontsize': 14}
-ax.set_xticklabels(["$r_{cut} - d$", "$r_{cut}$", "$r_{cut} + d$"], fontdict)
-ax.set_yticklabels(["$1.0$", "$0.5$", "$0$"], fontdict)
+ax.set_xticklabels([r"$r_{cut} - \frac{d}{2}$", r"$r_{cut}$",
+                    r"$r_{cut} + \frac{d}{2}$"], fontdict)
+ax.set_yticklabels(["$1.0$", "$0.5$", "$0$"])
```

Comment:
Sorry if this is too pedantic... can you revert the change on this line, so that it says `fontdict` and uses the larger font? The y-axis ticks were more legible the other way. Then this is good to merge.

Reply:
Let me know when this is fixed, it looks good otherwise so I can merge.

Reply:
It is changed.